_pages/index.md

```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
 
 ### Outreach
 
+* Appoint a storyteller for your project early in the cycle and notify #outreach. The storyteller will be your project's conduit with the outreach team and should notify the outreach team about: 
+
+    - upcoming milestones and events (that you’re going to or could present at)
+    - any impending risks to prepare for press and mitigate any issues
+    - progress on comms plan / blog posts
+    - any other ways the comms team can help your project
+    - upcoming speaking events / conferences that the project/product is represented at
+    - any updates that need to be conveyed to the agency partner’s comms team (and who those people might be.)
+    - updates for 18F newsletter or Dolores or #news
+    - updates for open source community who might be able to help with ‘help-wanted’ tags
```

Comment:
We add tasks that are requesting external support to our biweekly external newsletter to give them an additional boost. We'd like teams to let us know if there's an issue that we can surface to these external audiences.